approvaltests/namer/stack_frame_namer.py

```diff
@@ -51,10 +71,15 @@ def is_test_method(frame: FrameInfo) -> bool:
             and method_name != "_callTestMethod"
             and method_name != "run"
         )
+        return is_unittest_test
 
-        is_pytest_test = method_name.startswith("test_")
+    @staticmethod
+    def is_test_method(frame: FrameInfo) -> bool:
+        method_name = frame[3]
 
-        return is_unittest_test or is_pytest_test
+        return StackFrameNamer.is_unittest_test(
+            frame
+        ) or StackFrameNamer.is_pytest_test(method_name)
```

Comment:
it's bothering me that the StackFrameNamer.is_unittest_test and is_pytest_test take different arguments. I'd pass the frame to both.